django_enumfield/db/fields.py

```diff
@@ -54,6 +64,10 @@ def from_db_value(self, value, expression, connection, context):
 
         return value
 
+    def to_python(self, value):
+        if value is not None:
```

Comment:
to_python needs to handle the following cases:

* `value` is `None`
* `value` is a string
* `value` is an enum

ref https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#converting-values-to-python-objects